jaeger-core/src/test/java/io/jaegertracing/internal/SpanIdTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
```

Comment:
Please change year to `2018` seeing as this file is new. For other files that exist already, you can do `<year>-2018` based on what was there originally.